klayout/drc/testing/testcases/unit/mcell.svg

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
it's nice to see the failing test on the left and the passing test on the right!

is that a consistent convention across all rules, should we document it?

Reply:
can you comment here?

Reply:
We try to do this, but not always.